app/controllers/blogs_controller.rb

```diff
@@ -35,7 +35,6 @@ def edit
             @blog = @user.blogs.find_by(id:params[:id])
```

Comment:
there should be space in between `id:params[:id]`
make code readable

also, if there is any comment, see that whether this comment applies on any other place or not. Dont just change here, apply these at any other place which has similar issue.

Reply:
learn to use where, `@user.blogs.where(<query>)`
i have said before, find raises error
if you are using find, then use proper error handling